### PR TITLE
#714 - pypi proxy

### LINKED ISSUE #714: Update python and add proxy
Let's update python and add python proxy.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #714 
Updated pypi-adapter and added python proxy.

### CODE REVIEW

File: examples/pypi/README.md [hunk 1]
```diff
@@ -22,4 +22,26 @@ To install package with `pip install` specify Artipie repository url with `--ind
...
\ No newline at end of file
+```
+
+### Python proxy
+
+Try this `pypi-proxy.yaml` file to host a proxy to `https://pypi.org/simple/` repository:
+
+```yaml
+repo:
+  type: maven-proxy
```
Comment (olegmoz) [inline examples/pypi/README.md:33]: @olenagerasimova `type` value is invalid, please correct

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#716` is not assigned, can't get performer

**Pull request merged**